### ISSUE TITLE
Increased compartment pressure measurement

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10787]
**Preferred term label:**
Increased compartment pressure measurement

**Synonyms**

**Definition (free text, please give PubMed ID)**
Manometer (needle measurement) showing compartment pressure within 30 mmHg of diastolic blood pressure

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Compartment syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001172
- label: Increased compartment pressure
- parents: [Abnormal muscle physiology (HP:0011804)](https://hpo.jax.org/app/browse/term/HP:0011804)
- pmid: PMID:28846257
- def: intracompartmental pressure (ICP) over 30 mmHg. Acute compartment syndrome occurs when there is increased pressure within a closed osteofascial compartment, resulting in impaired local circulation. Acute compartment syndrome is considered a surgical emergency since, without proper treatment, it can lead to ischemia and eventually necrosis. Generally, acute compartment syndrome is considered a clinical diagnosis. However, ICP over 30 mmHg can be used as a threshold to aid in diagnosis.
- comment: Fascia is a thin, inelastic sheet of connective tissue that surrounds muscle compartments and limits the capacity for rapid expansion. The earliest objective physical finding is the tense, or ''wood-like" feeling of the involved compartment. Pain is typically severe, out of proportion to the injury. Measurement of intracompartmental pressure is not required but can aid in diagnosis if uncertainty exists. Compartment pressures are often measured with a manometer, a device that detects intracompartmental pressure by measuring the resistance that is present when a saline solution is injected into the compartment. The normal pressure within the compartment is between 0 mmHg to 8 mmHg. An intra-compartmental pressure greater than 30 mmHg indicates compartment syndrome and a need for fasciotomy.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10787
- synonyms:
